lib/internal/errors.js

```diff
@@ -707,6 +707,15 @@ E('ERR_INVALID_PERFORMANCE_MARK', 'The "%s" performance mark has not been set');
 E('ERR_INVALID_PROTOCOL', 'Protocol "%s" not supported. Expected "%s"');
 E('ERR_INVALID_REPL_EVAL_CONFIG',
   'Cannot specify both "breakEvalOnSigint" and "eval" for REPL');
+E('ERR_INVALID_SIGNATURE', (fn, signature, args) => {
+  const formatted = lazyUtil().inspect(args, {
+    depth: 0,
+    breakLength: Infinity
+  }).replace(/^\[\s*/, '(').replace(/\s*\]$/, ')');
```

Comment:
@BridgeAR I've tried that before and want to strip the spaces because `util.inspect` is not very consistent in that regard.

Reply:
In case you pass in the arguments as an array, it should be consistent?

Reply:
@BridgeAR Well..`util.inspect([])` returns `'[]'`, I could special casing for `args.length === 1` but IMO uses regex is more readable.

Reply:
That would only happen if `args.length === 0` as far as I see it. Because if you provide `[...arguments]` arguments have to be empty to result in `util.inspect([])`.

Reply:
I am not sure I can follow why you would want to add a special handling for `args.length === 0`. In that case the output with slicing would be `Received: child_process.execFile()`. And I thought that is what it should be?